external/harvestPackages/harvestPackages.netstandard1.6.depproj

```diff
@@ -1,146 +1,80 @@
-     <Version>4.3.0</Version>
-    </PackageReference>
-    <PackageReference Include="System.ValueTuple">
-     <Version>4.3.0</Version>
-    </PackageReference>
+    <LibraryPackage Include="Microsoft.CSharp" />
+    <LibraryPackage Include="Microsoft.VisualBasic" />
+    <LibraryPackage Include="Microsoft.Win32.Registry" />
+    <LibraryPackage Include="Microsoft.Win32.Registry.AccessControl" />
+    <LibraryPackage Include="System.Buffers" />
+    <LibraryPackage Include="System.Collections.Immutable" />
+    <LibraryPackage Include="System.Composition" />
+    <LibraryPackage Include="System.Composition.AttributedModel" />
+    <LibraryPackage Include="System.Composition.Convention" />
+    <LibraryPackage Include="System.Composition.Hosting" />
+    <LibraryPackage Include="System.Composition.Runtime" />
+    <LibraryPackage Include="System.Composition.TypedParts" />
+    <LibraryPackage Include="System.Data.SqlClient" />
+    <LibraryPackage Include="System.Diagnostics.DiagnosticSource" />
+    <LibraryPackage Include="System.IO.Compression" />
+    <LibraryPackage Include="System.IO.FileSystem.AccessControl" />
+    <LibraryPackage Include="System.IO.Packaging" />
+    <LibraryPackage Include="System.IO.Pipes.AccessControl" />
+    <LibraryPackage Include="System.Net.Http" />
+    <LibraryPackage Include="System.Net.Http.Rtc" />
+    <LibraryPackage Include="System.Net.Http.WinHttpHandler" />
+    <LibraryPackage Include="System.Numerics.Vectors" />
+    <LibraryPackage Include="System.Reflection.Context" />
+    <LibraryPackage Include="System.Reflection.DispatchProxy" />
+    <LibraryPackage Include="System.Reflection.Metadata" />
+    <LibraryPackage Include="System.Runtime.CompilerServices.Unsafe" />
+    <LibraryPackage Include="System.Runtime.InteropServices.RuntimeInformation" />
+    <LibraryPackage Include="System.Runtime.WindowsRuntime" />
+    <LibraryPackage Include="System.Runtime.WindowsRuntime.UI.Xaml" />
+    <LibraryPackage Include="System.Security.AccessControl" />
+    <LibraryPackage Include="System.Security.Cryptography.Cng" />
+    <LibraryPackage Include="System.Security.Cryptography.OpenSsl" />
+    <LibraryPackage Include="System.Security.Cryptography.Pkcs" />
+    <LibraryPackage Include="System.Security.Cryptography.ProtectedData" />
+    <LibraryPackage Include="System.Security.Cryptography.X509Certificates" />
+    <LibraryPackage Include="System.Security.Principal.Windows" />
+    <LibraryPackage Include="System.ServiceProcess.ServiceController" />
+    <LibraryPackage Include="System.Text.Encoding.CodePages" />
+    <LibraryPackage Include="System.Text.Encodings.Web" />
+    <LibraryPackage Include="System.Threading.AccessControl" />
+    <LibraryPackage Include="System.Threading.Tasks.Dataflow" />
+    <LibraryPackage Include="System.Threading.Tasks.Extensions" />
+    <LibraryPackage Include="System.ValueTuple" />
   </ItemGroup>
 
+  <Target Name="GetLatestStablePackages" BeforeTargets="RestorePackages">
```

Comment:
Also, you could avoid the static list by just looking at all PkgProj's in the repo and using their filename 😉